docs/tutorials/getting_started/intro.md

```diff
@@ -12,8 +12,6 @@ For a tutorial for older versions of Great Expectations, please see older versio
 
 The tutorial will walk you through the following steps:
 
-![minimap](../../images/minimap.png)
-
```

Comment:
Could we go ahead and delete `minimap.png` from the codebase if we no longer use it?

Reply:
Good catch. I've removed it.